src/Uno.UWP/ApplicationModel/Contacts/Contact.cs

```diff
@@ -0,0 +1,105 @@
+﻿#nullable enable
+
+using System.Collections.Generic;
+using System.Linq;
+using Windows.Foundation.Collections;
+
+namespace Windows.ApplicationModel.Contacts
+{
+	/// <summary>
+	/// Represents a contact.
+	/// </summary>
+	public partial class Contact
+	{
+		/// <summary>
+		/// Gets the full name of the Contact.
+		/// </summary>
+		public string FullName
+		{
+			get
+			{
+				return string.Join(" ", new [] { HonorificNamePrefix, FirstName, MiddleName, LastName, HonorificNameSuffix }.Where(n => !string.IsNullOrEmpty(n)));
```

Comment:
Not sure is this should be customized per culture. For instance in France it's pretty common to put the `LastName` (a.k.a. family name) first.

Definitely not a blocker, if we are able to easily (i.e. copy it :P ) fix that, it would be really nice.

Reply:
Hmm, you are right the order of the name parts should be culture dependent as per documentation, but not sure if there is some definitive guide on how to do it :-D .

Reply:
So, why implement this field? Erroneous implementation is much worse than no implementation.
Field should be left as unimplemented.
(I know, it is already merged...)